### ISSUE TITLE
[Bug]: Pressing enter key to toggle/untoggle one of the buttons in the find bar doesn't work

### ISSUE THREAD

**Issue by calixteman** [mozilla/pdf.js#19175]
### Attach (recommended) or Link to PDF file

Whatever pdf.

### Web browser and its version

Firefox nightly

### Operating system and its version

Windows 11

### PDF.js version

master

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

No

### Steps to reproduce the problem

- open the find bar
- tab to "Highlight all"
- press enter

### What is the expected behavior?

The button must be toggled.

### What went wrong?

It's still untoggled.

### Additional context

It works as expected with the space key.

**Comment by Snuffleupagus** [comment]
These DOM elements are `input type="checkbox"` and I *seems* to me that <kbd>Enter</kbd> not working might be a web-platform limitation since even in the Firefox *browser* findbar you can only check/uncheck the options with <kbd>Space</kbd>!?

**Comment by Snuffleupagus** [comment]
We could obviously work-around this ourselves, if that's desirable, with e.g. the following patch: https://github.com/mozilla/pdf.js/compare/master...Snuffleupagus:pdf.js:issue-19175

@calixteman Do you want me to submit a PR, or is this WONTFIX given https://github.com/mozilla/pdf.js/issues/19175#issuecomment-2520462803?

**Comment by calixteman** [comment]
Let me ask to UX/UI.

**Comment by calixteman** [comment]
@Snuffleupagus , in considering it isn't clear that it's a toggle button and it looks like a normal button, let's go for adding this keyboard shortcut.
